docs/troubleshooting.md

```diff
@@ -24,6 +37,79 @@ The Open vSwitch daemon logs for each `antrea-agent` Pod are also stored on the
 persistent storage of the corresponding node (i.e. the node on which the Pod is
 scheduled), under `/var/log/antrea/openvswitch`.
 
+## Accessing the antrea-controller API
+
+antrea-controller runs as a Deployment, exposes its API via a Service and
+registers an APIService to aggregate into the Kubernetes API. To access the
+antrea-controller API, you need to know the address of it and have credentials
+to access it. There are multiple ways you can do it.
+
+### Using antctl
+
+Typically, `antctl` handles locating and authenticating when it runs in an
+environment with kubeconfig set up. Same as `kubectl`, `antctl` looks for a
+file named config in the $HOME/.kube directory. You can specify other
+kubeconfig files by setting the `--kubeconfig` flag.
+
+For example, you can view internal NetworkPolicy with this command:
+```
+antctl get network-policy
+```
+
+### Using kubectl proxy
+
+As the antrea-controller API is aggregated into the Kubernetes API, you can
```

Comment:
suggestion to rephrase the first sentence:
As the antrea-controller API is aggregated into the Kubernetes API, you can access it through the Kubernetes API using the appropriate URL paths.

Reply:
done